pwiz_tools/Skyline/Model/Lib/BiblioSpec.cs

```diff
@@ -84,17 +84,20 @@ public static BiblioSpecLibSpec Deserialize(XmlReader reader)
     [XmlRoot("bibliospec_spectrum_info")]
     public sealed class BiblioSpecSpectrumHeaderInfo : SpectrumHeaderInfo
     {
-        public BiblioSpecSpectrumHeaderInfo(string libraryName, int spectrumCount, double? score, string scoreType)
+        public BiblioSpecSpectrumHeaderInfo(string libraryName, int spectrumCount, double? score, string scoreType, string protein)
             : base(libraryName)
         {
             SpectrumCount = spectrumCount;
             Score = score;
             ScoreType = scoreType;
+            _protein = protein;
         }
 
         public int SpectrumCount { get; private set; }
         public double? Score { get; private set; }
         public string ScoreType { get; private set; }
+        private string _protein;

```

Comment:
It's this way because the parent class declares Protein as read only: 
    public abstract string Protein { get; }
so private set isn't implied, and in fact isn't allowed.

So, we have to use a backing variable to be able to set the value during XML deserialization.

Reply:
Maybe a comment, then. Strange, since interfaces do allow { get; private set; } on a property with only { get; } in the interface. You would think an abstract would act like an interface.

Reply:
Will do, thanks.